homeassistant/components/ihc/__init__.py

```diff
@@ -329,30 +343,39 @@ def get_discovery_info(component_setup, groups, controller_id):
     return discovery_data
 
 
-def setup_service_functions(hass: HomeAssistantType, ihc_controller):
+def setup_service_functions(hass: HomeAssistantType):
     """Set up the IHC service functions."""
 
+    def __get_controller(call):
```

Comment:
A single underscore is enough.
```suggestion
    def _get_controller(call):
```